src/cgame/cg_rocket_draw.cpp

```diff
@@ -2376,6 +2376,50 @@ class SpawnQueueElement : public TextHudElement
 	int position;
 };
 
+class NumSpawnsElement : public TextHudElement
+{
+public:
+	NumSpawnsElement( const Rml::String& tag ) :
+			TextHudElement( tag, ELEMENT_DEAD ),
+			spawns( -1 ) {}
+
+	void DoOnUpdate() override
+	{
+		if ( !( cg.snap->ps.pm_flags & PMF_QUEUED ) || cg.snap->ps.pm_type == PM_DEAD )
+		{
+			if ( IsVisible() )
+			{
+				this->spawns = -1;
+				SetProperty( Rml::PropertyId::Visibility, Rml::Property( Rml::Style::Visibility::Hidden ) );
+			}
+			return;
+		}
+
+		int newSpawns   = cg.snap->ps.persistant[ PERS_SPAWNQUEUE ] & 0x000000ff;
+		if ( newSpawns == this->spawns )
+		{
+			return;
+		}
+		this->spawns = newSpawns;
+		if ( this->spawns == 1 )
+		{
+			SetText( _( "There are no spawns remaining" ) );
+		}
+		else
+		{
+			SetText( va( P_( "There is %d spawn remaining", "There are %d spawns remaining", this->spawns ), this->spawns ) );
+		}
+
+		if ( !IsVisible() )
+		{
+			SetProperty( Rml::PropertyId::Visibility, Rml::Property( Rml::Style::Visibility::Visible ) );
+		}
+	}
+
+	int spawns;
```

Comment:
We decided to go with trailing underscore for class members. https://wiki.unvanquished.net/wiki/Coding_convention#Naming_conventions

Reply:
Ah I didn't realize that. I was sticking to the Unv convention of no identifier for no class members :D.

Will fix and merge later.

Reply:
Fort he record I don't think this is a problem because it is consistent with the rest of the file.

Reply:
Done.